apps/inbound-mail/src/server/logger.ts

```diff
@@ -0,0 +1,62 @@
+import * as _ from 'lodash';
+import util from 'util';
+import * as winston from 'winston';
+
+const logger = new winston.Logger({
+  transports: [
+    new winston.transports.Console({
+      colorize: true,
+      prettyPrint: true,
+    }),
+  ],
+});
+
+logger.setLogFile = function (logFilePath) {
+  this.add(winston.transports.File, {
+    filename: logFilePath,
+    json: false,
+    maxsize: 20000000,
+    timestamp: true,
+  });
+};
```

Comment:
Let's write logs only to stdout for now until @Cliftonz will add the logger support on the infra level